src/main/resources/application.yml

```
@@ -18,7 +18,7 @@ spring:
     username: ${RABBIT_USERNAME}
     password: ${RABBIT_PASSWORD}
 application:
-  sparql-uri: ${FDK_BASE_URI:https://data.norge.no}/sparql
+  sparql-uri: ${FDK_SPARQL_SERVICE_URI:https://sparql.fellesdatakatalog.digdir.no}/fuseki/harvested/query
```

Den virker feil. Jeg er ganske sikker på at du ikke trenger `/fuseki/harvested/query` - det er intern konfig

```suggestion
  sparql-uri: ${FDK_SPARQL_SERVICE_URI:https://sparql.fellesdatakatalog.digdir.no}
```